examples/with-graphql-react/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "next-graphql-react-example",
+  "private": true,
+  "license": "ISC",
+  "dependencies": {
+    "cross-fetch": "^3.0.0",
+    "graphql-react": "^6.0.0",
+    "next": "^7.0.0",
```

Comment:
This should probably use `latest` like the other examples.

Reply:
At the time I surveyed the examples and noticed about 17 don't use `latest` for the `next` dependency. This is a good thing, since examples are not guaranteed to work when (not if) breaking changes to `next` are published in a major version.

There are almost no valid use cases for `*` or `latest`. Inexperienced people that download and run examples locally are likely to use them as a boilerplate for their own projects, and they will not know that `latest` is risky anti-pattern that *will* eventually blow up their project.

When the next major version of `next` is published I will be sure to publish a new version of `next-graphql-react` that supports in, with an updated peer-dependency range, and update the dependency in the example here.

Reply:
Seeing the major version of `next` supported by each example in `package.json` is also useful, as when a major new version is published not every example is guaranteed to work on day 1. It lets you know how up to date the example and the third-party projects are.